src/test/java/org/jenkinsci/plugins/credentialsbinding/impl/BindingStepTest.java

```diff
@@ -321,6 +321,25 @@ public void widerRequiredContext() throws Throwable {
         });
     }
 
+    @Issue("JENKINS-72412")
+    @Test public void maskingOfOneCharSecretShouldNotMangleOutput() throws Throwable {
+        rr.then(r -> {
+            String credentialsId = "creds";
+            String secret = "a";
```

Comment:
Uh, OK. Clearly this would not count as a “strong password”!

Reply:
I agree. I came across this unintentionally due to credentials for GitHub App, as you already know.